com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -115,7 +114,7 @@ private enum ComponentVersions
         [FormerlySerializedAs("m_LightVolumeOpacity")]
         [SerializeField] float m_LightVolumeIntensity = 1.0f;
         [SerializeField] bool m_LightVolumeIntensityEnabled = false;
-        [SerializeField] int[] m_ApplyToSortingLayers = new int[1];     // These are sorting layer IDs. If we need to update this at runtime make sure we add code to update global lights
+        [SerializeField] List<int> m_ApplyToSortingLayers;  // These are sorting layer IDs. If we need to update this at runtime make sure we add code to update global lights
```

Comment:
why is this a list? It doesn't change during runtime

Reply:
To perform LINQ operations. This has no runtime performance impact as it is serialized once.

Reply:
pretty sure most of linq applies to int[] as well as List[] since they are both iterable, but i could be out of touch.

AND does this load properly for projects serialized with int[]?